recipes/pyode/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# defining package and version
+{% set name = "pyode" %}
+{% set version = "0.16" %}
+{% set soversion = "0.16.0" %}
+{% set sha256 = "4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/odedevs/ode/downloads/ode-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - add-rt-to-dotpc                # [linux]
+    - modify-dotpc-win               # [win]
+    
+build:
+  number: 0
+  skip: true                         # [win and py==27]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}          # stage 1, library
+    - cmake
+    - {{ compiler('c') }}            # stage 2, generate bindings
+    - pkg-config
+    - m2-patch                       # [win]
+
+  host:
+    - cython
+    - python
+
+  run:
+    - python
+
+test:
+  requires:
```

Comment:
No need for this `requires` section as `python` is already in the `run` requirements.

Reply:
ok, removed